packages/flame_tiled/pubspec.yaml

```diff
@@ -8,7 +8,8 @@ environment:
   flutter: ">=2.5.0"
 
 dependencies:
-  flame: ^1.0.0
+  flame:
+    path: ../flame/
```

Comment:
```suggestion
  flame: ^1.0.0
```

Reply:
This should not be on path, melos is handling all of those things now, check here: :slightly_smiling_face: 
https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md#environment-setup